### PR TITLE
Frontpage modifications, banner and xDai icon integration

### PR DESCRIPTION
**Pull request by anthonyoliai**
This PR adds the xDai icon to the front page of the TEC interface, and also adds a banner. Requested as in [here](https://github.com/TECommons/tec-interface/issues/3)

### CODE REVIEW

File: src/components/ConvictionBanner.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    leave: { height: 0 },
+    config: springs.smooth,
+  })
+
+  return transitions.map(({ item: visible, key, props }) => {
+    return (
+      visible && (
+        <div
+          key={key}
+          style={{ ...props, overflow: 'hidden' }}
```
Comment (vivianedias) [inline src/components/ConvictionBanner.js:45]: the height isnt being set by props! if you check the aragon demo, this element has one

File: src/components/ConvictionBanner.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          css={`
+            width: 100%;
+            background: linear-gradient(
```
Comment (vivianedias) [inline src/components/ConvictionBanner.js:49]: maybe use the colors from [here](https://github.com/TECommons/tec-branding-assets)

File: src/lib/token-utils.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ const KNOWN_TOKENS_FALLBACK = new Map([
   ],
 ])
 
-const LOCAL_TOKEN_ICONS = new Map([['TESTTEC', tokenIconSvg]])
+const LOCAL_TOKEN_ICONS = new Map([['xDAI', tokenIconSvg]])
```
Comment (vivianedias) [inline src/lib/token-utils.js]: maybe we could add support to xDai, but not remove TESTEC from the array? 

the function `getTokenIconBySymbol()` will return the token requested according to the symbol passed

**Review by vivianedias — COMMENTED** [review]
Just some feedback and questions :) 

Great addition :+1:

**Comment by sembrestels** [comment]
Very cool, thank you two @anthonyoliai and @vivianedias!

**Pull request merged**